perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/ValidatedEmail.java

```diff
@@ -38,6 +38,11 @@
 
 	private InputGroup widget;
 
+	@Override
+	public void setEnabled(boolean enabled) {
+		getTextBox().setEnabled(enabled);
```

Comment:
Enabled/disabled state must be reflected also on email select, which is used to pick pre-filled emails and changes textbox content: `getSelect().setEnabled(enabled);`

Reply:
Done.